Ports/halflife/patches/ReadMe.md

```diff
@@ -0,0 +1,12 @@
+# Patches for Half Life on SerenityOS
+
+## `fix_mainui_wscript.patch`
+
+mainui_cpp-master/wscript: Link against libdl, libxml2
+When building support for menus in vgui, waflib links against
+fontconfig and freetype2. Unfortunately for us, it does so by
+uses the host system's pkgconfig to determine what to link
+against, which stops it from linking against libdl and libxml2
+(which are needed as transitive dependencies for fontconfig and
+freetype2 on serenity). This patch hacks in a check and build 
```

Comment:
this should be avoidable by setting `$PKG_CONFIG_PATH` to point into the sysroot

Reply:
Oh wow, I was not aware we had a working pkgconfig system. Yeah, this works after fixing the fontconfig port a bit.